app/js/app.js

```diff
@@ -94,38 +94,40 @@ window.showToast = function(message) {
 
 // Register service worker if supported.
 if ('serviceWorker' in navigator) {
-  navigator.serviceWorker.register('/service-worker.js'
-  ).then(function(registration) {
-    registration.onupdatefound = function() {
-      // The updatefound event implies that registration.installing is set; see
-      // https://slightlyoff.github.io/ServiceWorker/spec/service_worker/index.html#service-worker-container-updatefound-event
-      const installingWorker = registration.installing;
-      installingWorker.onstatechange = function() {
-        switch (installingWorker.state) {
-          case 'installed':
-            if (!navigator.serviceWorker.controller) {
-              window.showToast('Service Worker installed! Pages you view are cached for offline use.');
-            }
-            break;
-
-          case 'redundant':
-            throw Error('The installing service worker became redundant.');
+  window.addEventListener('load', function() {
```

Comment:
Perf - in the old timeline, the sw is loaded at the same time as the lazy imports (which should have priority over sw).